demos/README.md

```diff
@@ -8,6 +8,7 @@ The Open Model Zoo includes the following demos:
 - [Action Recognition Python* Demo](https://github.com/opencv/open_model_zoo/tree/master/demos/python_demos/action_recognition/README.md) - Demo application for Action Recognition algorithm, which classifies actions that are being performed on input video.
 - [Crossroad Camera C++ Demo](https://github.com/opencv/open_model_zoo/tree/master/demos/crossroad_camera_demo/README.md) - Person Detection followed by the Person Attributes Recognition and Person Reidentification Retail, supports images/video and camera inputs.
 - [Human Pose Estimation C++ Demo](https://github.com/opencv/open_model_zoo/tree/master/demos/human_pose_estimation_demo/README.md) - Human pose estimation demo.
+- [Gaze Estimation C++ Demo](https://github.com/opencv/open_model_zoo/tree/master/demos/gaze_estimation_demo/README.md) - Face detection followed by gaze estimation, head pose estimation and facial landmarks regression.
```

Comment:
G comes before H, so you should move this upwards.

Reply:
fixed